cmd/ebrelayer/txs/relayToEthereum.go

```diff
@@ -147,7 +161,9 @@ func initRelayConfig(provider string, registry common.Address, event types.Event
 	// Get the specific contract's address
 	target, err := GetAddressFromBridgeRegistry(client, registry, targetContract)
 	if err != nil {
-		log.Fatal(err)
+		log.Println(err)
+		return &ethclient.Client{}, &bind.TransactOpts{}, common.Address{}, err
```

Comment:
and here;

I feel like this line should be:
```
return nil, nil, common.Address{}, err
```
due to the function signiture using ponters

Reply:
Accepted.